templates/upload/process_id.html

```diff
@@ -49,11 +49,49 @@
                             {{ macros.render_field(form.notes) }}
                         </div>
                         <div class="medium-12 columns">
+                            {{ macros.render_field(form.report) }}
+                        </div>
+                        <div id="reportgithub">
+                        <div class="medium-12 columns">
+                            {{ macros.render_field(form.IssueTitle) }}
+                        </div>
+                        <div class="medium-12 columns">
+                            {{ macros.render_field(form.IssueBody) }}
+                        </div>
+                        <div class="medium-12 columns">
                             {{ macros.render_field(form.submit) }}
                         </div>
                     </div>
                 </form>
             </div>
         </div>
     </div>
-{% endblock %}
\ No newline at end of file
+{% endblock %}
+{% block scripts %}
+    {{ super() }}
+
+    <script type="text/javascript">
+        function report()
+        {
+            if($('#report').val()=='y')
+            {
+                $('#reportgithub').show();
+                $('#IssueBody').val('');
+                $('#IssueTitle').val('');
+            }
+            else
+            {
+                $('#reportgithub').hide();
+                $('#IssueBody').val('no-issue');
+                $('#IssueTitle').val('no-title');
+            }
+        }
+        $(document).ready(function()
+        {
+                $('#report').change(function() {
+                        report();
+                 });
+        });
+        report();
```

Comment:
Please call this inside `$(document).ready`, or there's a risk it'll fire before the DOM is fully loaded.